src/DataTable/IndeterminateCheckBox.jsx

```diff
@@ -7,7 +7,10 @@ const IndeterminateCheckbox = React.forwardRef(
     const resolvedRef = ref || defaultRef;
 
     React.useEffect(() => {
-      resolvedRef.current.indeterminate = indeterminate;
+      // this if(resolvedRef.current) prevents console errors in testing
+      if (resolvedRef.current) {
+        resolvedRef.current.indeterminate = indeterminate;
+      }
     }, [resolvedRef, indeterminate]);
```

Comment:
is the dependency `resolvedRef.current`?

Reply:
Yeah, the error I was getting was 
`Uncaught [TypeError: Cannot set property 'indeterminate' of null]`

Reply:
The ref is there, but `.current` is not.